<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 22, 22);
font: 14pt &quot;Gill Sans MT&quot;;
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="select_config">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1101</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
border-width :3px;
/*border-color: rgba(200, 200, 200,75);*/
border-radius : 15px;
border-style : solid;
	border-color: rgba(0, 0, 155, 150);
}
QGroupBox::title{
subcontrol-origin: margin;
   subcontrol-position: top center;

}

QStackedWidget{
border-width :2px;
border-color: rgba(200, 200, 200,75);
border-radius : 8px;
border-style : solid;
}</string>
    </property>
    <property name="title">
     <string>Konfigürasyon</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QStackedWidget" name="swPanel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(33, 33, 33,75);
</string>
     </property>
     <widget class="QWidget" name="add_panel">
      <widget class="QPushButton" name="b_add_panel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Pictures/Yeni klasör/panel button.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6"/>
    </widget>
    <widget class="QStackedWidget" name="swVentilation">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgba(33, 33, 33,75);
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="add_ventilation">
      <widget class="QPushButton" name="b_add_ventilation">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Pictures/Yeni klasör/ventilationbutton.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="choose_ventilation_unit">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>168</width>
         <height>37</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Motor Gücü</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbMotorRatedPower">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>0,25 Kw</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0,37 Kw</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0,55 Kw</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0,75 Kw</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1,1 Kw</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1,5 Kw</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>168</width>
         <height>37</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Fan adet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbMotorQuantity">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="bChooseVentilation">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ekle</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>168</width>
         <height>37</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Motor Voltajı</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbMotorRatedVoltage">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>380 V</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="swHeater">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(33, 33, 33,75);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="add_heater">
      <widget class="QPushButton" name="b_add_heater">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Pictures/Yeni klasör/HEATERbutton.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_12"/>
    </widget>
    <widget class="QStackedWidget" name="swCooler">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(33, 33, 33,75);</string>
     </property>
     <widget class="QWidget" name="add_cooler">
      <widget class="QPushButton" name="b_add_cooler">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Pictures/Yeni klasör/COOLERbutton.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_14"/>
    </widget>
    <widget class="QStackedWidget" name="swHumidity">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(33, 33, 33,75);</string>
     </property>
     <widget class="QWidget" name="add_">
      <widget class="QPushButton" name="b_add_humidity">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Pictures/Yeni klasör/HUMIDITYbutton2.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_16"/>
    </widget>
    <widget class="QStackedWidget" name="swPressure">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>30</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(33, 33, 33,75);</string>
     </property>
     <widget class="QWidget" name="add_1">
      <widget class="QPushButton" name="b_add_pressure">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/Pictures/Yeni klasör/PRESSUREbutton2.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>180</width>
         <height>311</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_17"/>
    </widget>
   </widget>
   <widget class="QLabel" name="laTotalPart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Toplam Parça</string>
    </property>
   </widget>
   <widget class="QLabel" name="laEstimatedPrice">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Yaklaşık Fiyat</string>
    </property>
   </widget>
   <widget class="QLabel" name="laDigitalInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dijital Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="laDigitalOutput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dijital Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTotalPart">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEstimatedPrice">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDigitalInput">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDigitalOutput">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerateBOM">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BOM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionYeni"/>
    <addaction name="actionA"/>
    <addaction name="actionKaydet"/>
    <addaction name="separator"/>
    <addaction name="actionKapat"/>
   </widget>
   <widget class="QMenu" name="menuEkle">
    <property name="title">
     <string>Ekle</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDuzenle">
    <property name="title">
     <string>Düzenle</string>
    </property>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuEkle"/>
   <addaction name="menuDuzenle"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionYeni">
   <property name="text">
    <string>Yeni</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>Aç</string>
   </property>
  </action>
  <action name="actionKaydet">
   <property name="text">
    <string>Kaydet</string>
   </property>
  </action>
  <action name="actionKapat">
   <property name="text">
    <string>Kapat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
